specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json

```diff
@@ -707,6 +707,10 @@
             "type": "string"
           }
         },
+        "isAlertable": {
```

Comment:
We try to stay away from booleans in Swagger and use string enums as they're extensible and forward compatible with new values. 
Alertable isn't a word in english and the term 'is' doesn't bring any value in this case. I'd suggest making this more specific to how its used with something like this:

'{serviceSpecificNameForUses}': ['alertCondition']

alertCondition may be the only thing allowed in this array for now but I'd assume this could grow.

This also constitutes as a breaking change and requires a new api-version, see 'New property added to response':
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo